lib/net/peer.js

```diff
@@ -137,11 +138,15 @@ class Peer extends EventEmitter {
    * Create inbound peer from socket.
    * @param {PeerOptions} options
    * @param {net.Socket} socket
+   * @param {Boolean?} brontide
    * @returns {Peer}
    */
 
-  static fromInbound(options, socket) {
```

Comment:
Maybe expose `PeerOptions` and construct in `Pool`, actually pass in `PeerOptions` to these functions. This will eliminate the need to have the `brontide` arg since it can exist in `PeerOptions`.